apps/mobile-wallet/locales/en-US/translation.json

```diff
@@ -437,5 +437,6 @@
   "nb_of_hidden_assets_one": "{{ count }} hidden asset",
   "nb_of_hidden_assets_other": "{{ count }} hidden assets",
   "Hide asset": "Hide asset",
-  "Show details": "Show details"
+  "Show details": "Show details",
+  "Hiding ALPH is not allowed": "Hiding ALPH is not allowed"
```

Comment:
I'm not sure I like the formulation "not allowed". WDYT about simply not showing the hide option for ALPH?

Reply:
agreed

Reply:
fixed in 0c10dfd27